fastlane/Fastfile

```diff
@@ -1,56 +1,141 @@
 # frozen_string_literal: true
 
+require 'dotenv'
+
 default_platform(:ios)
 
 UI.user_error!('Please run fastlane via `bundle exec`') unless FastlaneCore::Helper.bundler?
 
 PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
 APP_FOLDER = File.join(PROJECT_ROOT_FOLDER, 'ios', 'App')
+
 BUNDLE_IDENTIFIER = 'com.ellavandurpe.blocknotes'
 
+APP_NAME_IOS = 'blocknotes-ios'
+APP_NAME_MAC_CATALYST = 'blocknotes-mac-catalyst'
+
+# Note that we cannot make this a subfolder of APP_FOLDER, or 'npx cap sync' will fail with:
+#
+# ✔ Updating iOS plugins in 1.60ms
+# ✖ Updating iOS native dependencies with bundle exec pod install - failed!
+# ✖ update ios - failed!
+# [error] Command line invocation:
+#         /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -project
+#         App.xcodeproj clean
+
+#         User defaults from command line:
+#         IDEPackageSupportUseBuiltinSCM = YES
+
+#         error: Could not delete `<APP_FOLDER>/build`
+#         because it was not created by the build system.
+OUTPUT_FOLDER = File.join(PROJECT_ROOT_FOLDER, 'build')
+
+ENV_FILE_NAME = 'blocknotes.env'
+USER_ENV_FILE_PATH = File.join(Dir.home, '.a8c-apps', ENV_FILE_NAME)
```

Comment:
I really like this convention! I have a _lot_ of different env files in my home directory and it would be great to organize them 😄